athena/data/datasets/speech_recognition_kaldiio.py

```diff
@@ -93,6 +96,8 @@ def __getitem__(self, index):
         feat = self.kaldi_io_feats[key]
         feat = feat.reshape(feat.shape[0], feat.shape[1], 1)
         feat = tf.convert_to_tensor(feat)
+        if self.hparams.spectral_augmentation is not None:
+            feat = self.spectral_augmentation(feat)
```

Comment:
Maybe we should apply specaug after finishing feature normalization?

Reply:
I think you are right, after feature norm is better for online cmvn